spec/integration/application/agent_spec.rb

```diff
@@ -242,4 +243,109 @@
       end
     end
   end
+
+  context 'https file sources' do
+    let(:path) { tmpfile('https_file_source') }
+    let(:response_body) { "from https server" }
+    let(:digest) { Digest::SHA1.hexdigest(response_body) }
+
+    it 'rejects HTTPS servers whose cert is not issued by the puppet CA' do
```

Comment:
maybe clearer to say "not in the system CA store"?

Reply:
Yes, this test title is a bit misleading

Reply:
Updated